_posts/2015-10-30-october-newsletter.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: newsletter
+title: "Newsletter #6 - Ship it!"
+category: newsletter
+permalink: /news/2015/october/
+---
+
+Welcome to the sixth newsletter for Neovim, a project that aims to improve Vim
+by adding [new features][terminal-emulator] and wrap it all in a nice, modern
+face.
+
+### Introduction
+
+Hi, this is @tarruda and I will be addressing the Neovim community directly in
+this newsletter. Other than that, I will try to keep it structured as @jdavis
+did previously. Let's get started!
+
+## General News
+
+### 0.1 release
+
+Neovim now has its first public release!
+
+A few months ago, @justinmk created the [0.1 milestone][0.1-milestone] which
+greatly helped us focus on more urgent tasks that resulted in the first release.
+We planned many features not yet available in 0.1, but decided to postpone them
+for future milestones, which will be more frequent after this newsletter.
+
+This illustrates the path Neovim will take from now on: Instead of preparing
+big releases that take forever to happen, we'll focus on smaller, frequent and
+more stable releases.
+
+The 0.1 release is basically just a tag for users looking to compile Neovim in
+a version that has a minimum level of stability, but future releases may also
+contain precompiled binaries and even installers (when Windows is officially
+supported).
+
+Neovim 0.1 is already available for [Homebrew/Linuxbrew][homebrew-formula] and
+[Arch Linux][archlinux-pkgbuild]. Check the
+[installation page on the Neovim Wiki][installing-neovim-package] for more
+possibilities to install Neovim (although, at the time of writing, most of these
```

Comment:
Maybe link to the "Building Neovim" page too, since many platforms on said page are not mentioned on "Installing Neovim".